lib/tools/system-calls.js

```diff
@@ -727,20 +721,27 @@ systemCallMethods.checkAvdExist = async function (avdName) {
  * @throws {Error} If the emulator is not ready within the given timeout.
```

Comment:
Return value should be documented.

Reply:
the method always returns undefined

Reply:
Really? Why are lines 728 and 736 returning `false`?